src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -404,6 +418,42 @@ def _get_linux_fx_version(cmd, resource_group_name, name, slot=None):
     return site_config.linux_fx_version
 
 
+def url_validator(url):
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc, result.path])
+    except:
+        return False
+
+
+def _get_linux_multicontainer_config_file(cmd, resource_group_name, name, file_name=None, encoded=True, slot=None):
+    from base64 import b64encode, b64decode
+    config_file_bytes = None
+    if file_name:
+        if url_validator(file_name):
+            from urllib.request import urlopen
+            response = urlopen(file_name)
```

Comment:
this will not work across platform, particularly windows and linux, as you need to set the SSL context. Please check out how [acs does](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py#L92)
Or, for this version, just pull out the support for opening from url